MFVNeutralino/python/EventFilter.py

```diff
@@ -128,7 +128,7 @@ def setup_event_filter(process,
                     process.load('JMTucker.Tools.UnpackedCandidateTracks_cfi')
                     process.mfvVertexTracks.tracks_src = 'jmtUnpackedCandidateTracks'
                     process.jmtRescaledTracks.tracks_src = 'jmtUnpackedCandidateTracks' # JMTBAD use rescaled tracks
-            vertexFilter = cms.EDFilter('VertexSelector', src = cms.InputTag('mfvVertices'), cut = cms.string('nTracks > 2'), filter = cms.bool(True))
+            vertexFilter = cms.EDFilter('VertexSelector', src = cms.InputTag('mfvVertices'), cut = cms.string('nTracks > 2'), filter = cms.bool(False)) #PK: used to be True
```

Comment:
This also shouldn't need to be changed

Reply:
I remember this one needs to be False so that the vertexer won't throw an error about pat Jets but I tried running a small batch locally with this option turned on and it seemed to run just fine. So, I think I agree with you.

Reply:
DONE